tests/script-based-pre/zero-size/config.yml

```diff
@@ -0,0 +1,4 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+script: playback_opts.sh
```

Comment:
nit: playback_zst?

Also, can you please name the folder playback-zero-size?

Reply:
277a15967fe578894eda404be4c3b3751145c0c5